website/docs/reference/warehouse-setups/dremio-setup.md

```diff
@@ -76,11 +79,6 @@ Before connecting from project to Dremio Cloud, follow these prerequisite steps:
     * `dremio_cloud` for working with Dremio Cloud
     * `software_with_username_password` for working with a Dremio Software cluster and authenticating to the cluster with a username and a password
     * `software_with_pat` for working with a Dremio Software cluster and authenticating to the cluster with a personal access token
-4. Append these lines to the end of the content of the `dbt_project.yml` file at the root of your project directory:
-```
-vars:
-    dremio:reflections_enabled: false
-```
```

Comment:
is this setting no longer valid?

Reply:
It is no longer necessary and we changed default behaviour from being reflections enabled to now being false. The reason for this change is that you need to be an administrator in Dremio to enable reflections.

We are working on supplementary documentation to explain how reflections work with dbt and are targeting for publishing this next week.

The supplementary documentation is based on the README from the community connector that has been tested for accuracy and currency, and cleaned up fore readability.